trackstop.lua

```diff
@@ -236,7 +260,170 @@ function RollerOverlay:init()
   }
 end
 
+selected_stop = selected_stop or nil
+
+ReorderStopsWindow = defclass(ReorderStopsWindow, widgets.Window)
+ReorderStopsWindow.ATTRS {
+  frame={t=4,l=60,w=45, h=28},
```

Comment:
since this window contains a list with a variable number of elements, it would be useful to set `resizable=true`